modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -218,6 +180,49 @@ enum ImwriteHDRCompressionFlags {
 
 //! @} imgcodecs_flags
 
+/** @brief Represents an animation with multiple frames.
+The Animation struct is used to store and manage the data for an animated sequence.
+It includes information about the number of times the animation should loop, the background color,
+timestamps for each frame, and the frames themselves.
+*/
+struct CV_EXPORTS_W_SIMPLE Animation
+{
+    //! Number of times the animation should loop. 0 means infinite looping.
+    CV_PROP_RW int loop_count;
+    //! Background color of the animation in RGBA format.
+    CV_PROP_RW int bgcolor;
```

Comment:
M.b. `cv::Scalar`? We use it everywhere for the same purpose. There is no issue with big-little endianess.